blueprint/src/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "base-component",
+  "description": "oc-template-react base component",
+  "version": "0.0.1",
+  "repository": "https://github.com/opentable/oc-template-react",
```

Comment:
I think I would remove repository? Just to ensure people don't use this by mistake